eqcorrscan/tests/lag_calc_test.py

```diff
@@ -170,11 +172,20 @@ def test_lag_calc_api(self):
             detections, self.data, template_names, templates,
             shift_len=0.2, min_cc=0.4, horizontal_chans=['E', 'N', '1', '2'],
             vertical_chans=['Z'], cores=1, interpolate=False,
-            plot=False, npy=False)
+            plot=False, export_cc=False)
         self.assertEqual(len(output_cat), len(detections))
         for event in output_cat:
             self.assertEqual(len(event.picks), len(self.data))
 
+    def test_xcorr_pick_family_export_cc(self):
+        cc_dir = 'cc_exported'
+        xcorr_pick_family(
+            family=self.party[0], stream=self.data, shift_len=0.2, plot=False,
+            interpolate=False, export_cc=True, cc_dir=cc_dir)
+        cc_files = glob.glob(os.path.join(cc_dir, '*.npy'))
```

Comment:
```suggestion
        cc_files = glob.glob(os.path.join(cc_dir, '*.npy'))
        assert len(cc_files) == len(self.party[0])
```
It looks like you expect one file to be written per detection - so you can just check that the number of files equals the number of detections.